Frontend/Pages/UserManagement/Register.razor

```diff
@@ -83,14 +87,28 @@
     </div>
 </div>
 
-@* TODO implement registration logic *@
 
 @code {
     readonly Registration _registration = new();
 
-    private void HandleValidSubmit() {
-        Console.WriteLine("Valid");
-        Console.WriteLine(_registration.Email);
+    private async void HandleValidSubmit() {
+        var request = new HttpRequestMessage(HttpMethod.Post,
+            "https://localhost:7038/api/register");
```

Comment:
Should be using relative addressing which is probably just "api/register" (check to make sure)

Reply:
Yes, when using the scoped HttpClient, see also comment in Program.cs